src/widgets/settingspages/CommandPage.cpp

```diff
@@ -38,18 +39,35 @@ CommandPage::CommandPage()
     LayoutCreator<CommandPage> layoutCreator(this);
     auto layout = layoutCreator.setLayoutType<QVBoxLayout>();
 
-    EditableModelView *view =
+    this->view =
         layout.emplace<EditableModelView>(app->commands->createModel(nullptr))
             .getElement();
 
-    view->setTitles({"Trigger", "Command", "Show In\nMessage Menu"});
-    view->getTableView()->horizontalHeader()->setSectionResizeMode(
+    this->view->setTitles({"Trigger", "Command", "Show In\nMessage Menu"});
+    this->view->getTableView()->horizontalHeader()->setSectionResizeMode(
         1, QHeaderView::Stretch);
-    view->addButtonPressed.connect([] {
+    this->view->addButtonPressed.connect([] {
         getApp()->commands->items.append(
             Command{"/command", "I made a new command HeyGuys"});
     });
 
+    QItemSelectionModel *selectionModel =
+        this->view->getTableView()->selectionModel();
+    QObject::connect(
+        selectionModel, &QItemSelectionModel::currentChanged, this,
+        [this](const QModelIndex &current, const QModelIndex &previous) {
+            if (this->checkCommandDuplicates())
+            {
+                this->duplicateCommandWarning->show();
+            }
+            else
+            {
+                this->duplicateCommandWarning->hide();
+            }
```

Comment:
Instead of checking the return value, set the visibility of `duplicateCommandWarning` at the bottom of the `checkCommandDuplicates` command